Modules/Loadable/VolumeRendering/Logic/vtkSlicerVolumeRenderingLogic.cxx

```diff
@@ -948,6 +948,18 @@ void vtkSlicerVolumeRenderingLogic::UpdateDisplayNodeFromVolumeNode(
 }
 
 //----------------------------------------------------------------------------
+void vtkSlicerVolumeRenderingLogic
+::UpdateDisplayNodeFromVolumeNode(vtkMRMLVolumeRenderingDisplayNode *paramNode,
+                                  vtkMRMLVolumeNode *volumeNode)
+{
+  vtkMRMLVolumePropertyNode *propNode =
+    paramNode ? paramNode->GetVolumePropertyNode() : NULL;
```

Comment:
Just from looking at the code it seems to me that Slicer will crash if paramNode is NULL (in the 4-argument UpdateDisplayNodeFromVolumeNode a method of paramNode is called, without checking first if paramNode is NULL or not). Add checking of vtkMRMLVolumeRenderingDisplayNode *paramNode/displayNode and immediately log error and return if it's NULL.